extensions/resteasy-reactive/quarkus-rest-client-microprofile/runtime/src/main/java/io/quarkus/rest/rest/client/microprofile/DefaultMicroprofileRestClientExceptionMapper.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.rest.rest.client.microprofile;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.ext.ResponseExceptionMapper;
+
+public class DefaultMicroprofileRestClientExceptionMapper implements ResponseExceptionMapper {
+
+    public Throwable toThrowable(Response response) {
+        try {
+            response.bufferEntity();
+        } catch (Exception var3) {
+        }
+
+        return new WebApplicationException("Unknown error, status code " + response.getStatus(), response);
+    }
+
+    public boolean handles(int status, MultivaluedMap headers) {
+        return status >= 400;
+    }
+
+    public int getPriority() {
+        return 2147483647;
```

Comment:
Maybe use `Integer.MAX_VALUE` to avoid magical number?

Reply:
done